src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
The class `Figure` is abstract and implements both `AreaCalculator` and `Drawable`. According to the checklist, abstract classes should not be used to set behavior for classes. Consider using interfaces to define behavior instead.